src/Package.php

```diff
@@ -43,6 +50,7 @@ class Package {
 	public static function init() {
 		$wordpress_minimum_met = version_compare( get_bloginfo( 'version' ), '5.3', '>=' );
 		if ( ! $wordpress_minimum_met ) {
+			self::$active_version = __( 'Inactive ', 'woocommerce-admin' );
```

Comment:
Should we leave this as `null` for comparison checks later?  I think the `Inactive` string should be set in core.  We may also need some other changes there as the language is a bit odd currently:

<img width="598" alt="Screen Shot 2020-03-10 at 11 39 00 AM" src="https://user-images.githubusercontent.com/10561050/76304681-142fa400-62c4-11ea-9818-2f850c87bf73.png">

Reply:
>Should we leave this as null for comparison checks later?

Probably a good idea. Changed in https://github.com/woocommerce/woocommerce-admin/pull/3851/commits/220d11540329441a6426f56a1efea707002fd44d